apps/ashley/templates/services/postgresql/deploy.yml.j2

```diff
@@ -44,7 +44,7 @@ spec:
           envFrom:
             - secretRef:
                 name: "{{ ashley_database_secret_name }}"
-{% endif%}
       securityContext:
         runAsUser: {{ container_uid }}
         runAsGroup: {{ container_gid }}
+{% endif%}
```

Comment:
Is this the only impacted template?

Reply:
Yes

Reply:
Uh, oh. I was sure I checked everything.
But I double checked and apparently my memories are not reliable :sweat_smile: 
I'll fix this.

Reply:
Done in 3d294c4b469b6997bb02bc441159dedbcc56b5ca